app/views/exam_templates/_exam_template_upload.html.erb

```diff
@@ -0,0 +1,7 @@
+<label>Upload Exam Template</label>
+<%= form_tag({:action=>"update", :controller=>"exam_templates", :id=>exam_template.id}, :multipart => true) do %>
+    <p>
+      <%= file_field_tag 'new_template' %>
```

Comment:
This actually shouldn't be a separate partial. Instead, this tag belongs in the form currently contained in `_exam_template_download` (whose contents should actually just go into `_form`)